src/resources/UsageAnalytics/Read/DataHealth/DataHealth.ts

```diff
@@ -93,4 +95,13 @@ export default class DataHealth extends ReadServiceResource {
     protected buildPathWithOrg(route: string, queryParams?: any): string {
         return super.buildPath(route, {org: this.api.organizationId, ...queryParams});
     }
+
+    /**
+     * Get failed instances for a data health criterion
+     */
+    getFailedInstances(params: DataHealthGetFailedInstancesParams, args?: RequestInit) {
+        return this.api.get<DataHealthGetFailedInstancesResponse>(
+            this.buildPathWithOrg(`${DataHealth.baseUrl}/criteria/failedInstances`, params),
```

Comment:
You accept `args` as an argument but you're not using it in the call, is the experimental client in the admin UI actually using `args`?

Reply:
The other calls aren't using `args` either, that's what the orange squiggly line in your IDE is telling you :P

Reply:
Yea they were superfluous, I removed all of them.